CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.8)
-set(CMAKE_OSX_DEPLOYMENT_TARGET "10.12" CACHE STRING "Minimum OS X deployment version")
+set(CMAKE_OSX_DEPLOYMENT_TARGET "10.15" CACHE STRING "Minimum OS X deployment version")
```

Comment:
because of the new Tor implementation, right now, some parts of the code are just supported on `10.15`;  I think; the minimum version required could be tackle when the code base reaches a stable release... what do you think?

Reply:
Let's try to get this resolved now, if we can, or postpone this one change. What failures do you get?

Reply:
![Screen Shot 2019-10-22 at 6 49 10 PM](https://user-images.githubusercontent.com/337107/67309760-efd95380-f4eb-11e9-840c-01080c8b0044.png)